src/Sulu/Bundle/MediaBundle/Resources/js/components/SingleMediaDropzone/singleMediaDropzone.scss

```diff
@@ -4,6 +4,7 @@ $dropzoneSize: 400px;
 $uploadIconColor: $scorpion;
 $uploadInfoTextColor: $scorpion;
 $uploadIndicatorBackgroundColor: rgba($alabaster, .8);
+$mediaContainerBreakpoint: 520px; /* dropzoneSize + 2 * $zviewPadding */
```

Comment:
Why don't you calculate the value as described in the comment using css?

Reply:
Because you can't use `calc` inside a `media` query. https://stackoverflow.com/questions/23667208/calc-not-working-within-media-queries?answertab=votes#tab-top

Reply:
Can you maybe put a code comment here? Otherwise it is just confusing